### PR TITLE
completed helm and added config utils

### PR DESCRIPTION
**Pull request by idoabelman**

### CODE REVIEW

File: helm/feed_reporter/templates/cronjob-feed-reporter.yaml [hunk 1]
```diff
@@ -10,20 +10,18 @@ spec:
         spec:
           containers:
           - name: feed-reporter
-            image: test/feed-reporter
-            imagePullPolicy: Never
+            image: eu.gcr.io/cyco-main/intel471-feed-reporter
```
Comment (shiranpoz) [inline helm/feed_reporter/templates/cronjob-feed-reporter.yaml:13]: plz add image tag and put it in values file. teamcity will update it after each build

Comment (shiranpoz) [reply helm/feed_reporter/templates/cronjob-feed-reporter.yaml:13]: image itself can be a value as well (it doesn't get set)

File: helm/feed_reporter/values.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+resources:
+  requests:
+    memory: 64m
+    cpu: 100m
+  limits:
+    memory: 0.5G
+    cpu: 120m
+
```
Comment (shiranpoz) [inline helm/feed_reporter/values.yaml:8]: add "tag" at top level (can be latest by def)

**Review by shiranpoz — CHANGES_REQUESTED** [review]

**Comment by jefimm** [comment]
Please make sure the helm/... is named helm/intel471_feed_reporter and not feed_reporter

**Review by jefimm — APPROVED** [review]

**Pull request merged**